geo/src/algorithm/line_measures/metric_spaces/euclidean/bearing.rs

```diff
@@ -0,0 +1,74 @@
+use crate::algorithm::Bearing;
+use crate::CoordFloat;
+use crate::Euclidean;
+use crate::Point;
+use num_traits::FromPrimitive;
+
+impl<F: CoordFloat + FromPrimitive> Bearing<F> for Euclidean {
+    /// Returns the bearing from `origin` to `destination` in degrees along a straight line.
+    ///
+    /// # Units
+    ///
+    /// - `origin`, `destination`: Points where x/y are cartesian coordinates
+    /// - returns: degrees, where: North: 0°, East: 90°, South: 180°, West: 270°
```

Comment:
I think this will often (usually) be correct, but I wonder if there is a more neutral way to phrase this that isn't predicated on having coordinates where +Y is due north and +X is due east.

Maybe something like:

```
/// # Units
///
/// - `origin`, `destination`: Points where x/y are cartesian coordinates
/// - returns: degrees, measured clockwise from the positive Y-axis direction
///   - 0° = positive Y direction (typically North)
///   - 90° = positive X direction (typically East)
///   - 180° = negative Y direction (typically South)
///   - 270° = negative X direction (typically West)
```

I feel like @urschrei might have some input.

Reply:
^^ This is clearer, IMO. Thanks @michaelkirk